website/content/docs/enterprise/license/faq.mdx

```diff
@@ -0,0 +1,78 @@
+
+No. Only Nomad server nodes need to have the license present.
+
+Nomad agents running with the “-dev” flag will also need a license (as they are running a server agent).
+
+## Q: What are some pitfalls an ops person/operator/SRE/etc could experiency by up an upgrade because of this?
+
+If you follow the steps in the [upgrade guide](https://www.nomadproject.io/docs/upgrade/upgrade-specific#enterprise-licenses), but have misconfigured the licensing steps, the server instances will stop the boot-up sequence. 
+
+If you do not check cluster health after adding new servers, and verify that the new servers have joined properly, then you may accidentally take down old servers without replacing them.
+
+If you are using a trial license in production, this could cause issues as the cluster will fail start once the trial expires.
+
+If you have built automation around `nomad license put`, this command will fail as it has been removed from Nomad v1.1.0+.
+
+## Q: What are some other gotchas?
+
+Operators at a company might not have access to the company's normal Nomad license. 
+The operators may be relying on the now-removed 6-hour window in order to do daily work with Nomad. 
+Running `nomad agent -dev` will fail for these users. In this case, they should get a trial license via the [trial license page](http://nomadproject.io/trial).
+
+## Q: How can a user get their existing license to put on disk for an upgrade?
+
+Use `nomad license get` to retrieve the license, then save the license to disk.
+
+## Q: What happens when a license expires?
+If a license expires after the server instance is joined to the cluster, the existing behavior is preserved: disallow Enterprise feature operations until a grace period expires, at which point the Nomad servers shut down. 
+The grace period will be one day.
+
+## Q: Will on-prem binaries with “baked-in” licenses work?
+
+Yes. They will work as if they had a standard autoloaded license.
+
+
+## Q: Are license files cluster-specific or node-specific?
+
+No. Licenses can be reused on different nodes and different clusters.
+
+## Q: Where can users get a trial license for Nomad?
+Visit http://nomadproject.io/trial for a free 30-day trial license.
+
+## Q: When a Nomad client agent restarts, will it still need a license file?
+
+Yes. Every time `nomad agent` runs as a server (or in -dev), it requires a license on disk.
+
+## Q: Will Nomad downgrades/rollbacks work?
+
+When downgrading to a version of Nomad before 1.1.0, customers will need to follow the old  process for applying enterprise licenses to Nomad. 
+This is detailed in the [0.12 portion of the upgrade guide](https://www.nomadproject.io/docs/upgrade/upgrade-specific#enterprise-licensing). Enterprise features will be enabled for 6 hours. 
+During that time enterprise users should apply their license with the “nomad license put” command. 
+Licenses will not be autoloaded from the file system in earlier versions of Nomad.
```

Comment:
Nomad cannot be downgraded at all. "Autoloaded" is not a user-facing concept.

Reply:
@tgross Can you elaborate on this a bit more?  The statement about Nomad not being down gradable is not making sense. Autoloaded is part of the boot-up sequence and outside the consumers control, but they are still available to roll back to a prior version of Nomad Enterprise.

Reply:
There's two separate things I'm saying here:
* You cannot safely roll back to an earlier version of Nomad at all, licensing aside. It's not a supported operation. The Nomad state store has no guarantee of forward compatibility, so trying to roll back to an earlier version can result in a crash loop. And Nomad cannot be downgraded from ENT to OSS safely either ([example](https://github.com/hashicorp/nomad/issues/9774#issuecomment-772655733))
* "Autoloaded" is not a user-facing concept, by which I mean the term "autoloaded" is not in our documentation at all. That term is only used in the internal design documents. So it doesn't mean anything to tell a user that a license will not be "autoloaded". The language here should match the user-facing documentation.

Reply:
I see, so perhaps adding something along these lines. 

Q: Can I downgrade Nomad Enterprise from v1.1.0 to an earlier version?
Nomad cannot safely be rolled back to an earlier version. This is not a supported operation due to the Nomad state store not having a forward compatibility. The end result could potentially be a crash loop.